portal/v2/src/ClusterList.tsx

```diff
@@ -255,6 +255,20 @@ class ClusterListComponent extends Component<ClusterListComponentProps, ICluster
               />
             </TooltipHost>
             <KubeconfigButton resourceId={item.resourceId} csrfToken={props.csrfToken} />
+            {/* <TooltipHost content={`Geneva`}>
```

Comment:
This would be something which we might add in future currently we do not have something at backend to handle this. Whenever we would have the a backend for this we would just have to uncomment this, so I think let the comment be there.

Reply:
most of the time commented out code left for the future is forgotten about, do we have an ADO card for this future feature?

Reply:
I don't think so if we have an ADO card for this. What I can do is, I can just remove the comment for now and talk to Amber about the timeline of adding this and then would create an ADO card for the same. Sounds Good?

Reply:
Sounds like a good plan to me, thank you!